README.md

```diff
@@ -44,49 +44,48 @@ Your code interview will be evaluated based on your repo, so make sure all files
 
 We love the NYC 311 service and the open data products that come with it. In this challenge, you will use **[311 Service Requests from 2010 to Present](https://data.cityofnewyork.us/Social-Services/311-Service-Requests-from-2010-to-Present/erm2-nwe9)** on NYC Open Data, write an ETL pipeline, and produce some data insight.
 
-### Task 1: Data Download
+This challenge has two halves, a python/bash half with 4 steps and a docker/SQL part with 2 steps. The second half will use a file generated by the first python step. It's our expectation that the logic to perform each step in the first half of this challenge will be written in python, and be callable by an executable script that takes specified arguments as inputs.
 
-Write a python script/notebook to download all service request records created in the **last week** (7 days) and has **HPD** as the responding agency, and store the data in a csv named `raw.csv` in a folder called `data`.
+### Python Task 1: Data Download
 
-### Task 2: Data Aggregation
+To download 311 service request records write a script that takes 2 parameters passed from the command line: number of days and responding agency acronym. For example, if a user wanted to get all service request records created in the last week where DSNY is the responding agency, they would pass `7` and `DSNY` as the parameters. For this exercise, we ask that you download all 311 service requests filed the **last seven days** where **HPD** is the responding agency.  Save the data as a csv named `raw.csv` in a folder called `data`. 
```

Comment:
The reason I would prefer to have them choose the number of days and responding agency is that it has them read the source data themselves and see what the responding agencies are. Having them save multiple files with names of their choice tests their ability to cache well-named files. I would prefer to see `HPD_last_7.csv` and `DOT_last_10.csv` with the filenames constructed by the python code rather than `data1.csv` and `data2.csv`.

Additionally, if we ask them to read the whole challenge before starting they will know not to chose one day or 1,000 days as these don't produce such good plots

Reply:
@SashaWeinstein That makes sense to me and tests their data acumen but can see @AmandaDoyle point as well